Chapter 12 (Final Project) revision based on reading session (#31)
GBKS/opendesign.guide: content/12-first-project.md

1. **Create a pull request or equivalent:** Follow the project's process for submitting contributions.
1. **Be responsive:** Answer questions and be open to making revisions based on community feedback.
1. **Help with implementation:** If possible, assist the developers in implementing your design.
1. **Follow up:** How was the response to your changes? Are users happier now?

Your presentation should be clear, professional, and show how your work benefits the project.

::resources
- [Feedback request - Ideas for improving how multiple accounts are handled in the GUI](https://www.reddit.com/r/Monero/comments/c23zxn/feedback_request_ideas_for_improving_how_multiple/) via Monero
::

## Step 6: Reflection and Documentation

After your contribution is accepted (or even if it isn't), take time to reflect:

1. **Identify lessons learned:** Review your notes. What went well? What would you do differently next time?
1. **Write a case study:** Document the entire process from start to finish. This is great for your portfolio and helpful for other designers.

> 1. **Share your experience:** Consider writing a blog post or giving a talk about what you learned.

Reviewer: ```suggestion
- **Share your learnings:** Consider sharing your experience with the wider community. This could be through a blog post, social media thread, or even a short Loom video. It’s a way to give back to the community while building your credibility.
```

## Project Timeline

It can be helpful to have a timeline in mind as you start out. Due to how organically many open-source projects collaborate, assume that you may need to adjust it as you go. Here's a simple one you use as a foundation:

- Weeks 1-2: Choose a project and understand it
- Week 3: Propose your contribution
- Weeks 4-6: Do the design work
- Week 7: Present your work and handle feedback
- Week 8: Reflection and documentation

You have most control over your own work timeline, and least control over the parts where you rely on others (like feedback and implementation). Make the most of these dynamics, and as always, clear and regular communication is the key.

## Mentor Evaluation

If you would like feedback, you could ask a mentor or another designer for an evaluation. Here are some criteria they may be looking for:

1. **Choice of project:** Was it a good fit for you and your interests and skillset?
1. **Understanding of the project:** Did you grasp the project's needs and context?

> 1. **Quality of your design work:** Does it solve a real problem effectively?

Reviewer: ```suggestion
1. **Design Quality:** Does your design solve a real problem effectively and improve the user experience?
```

1. **Community interaction:** How well did you communicate and handle feedback?
1. **Documentation:** How well did you explain your process and decisions?

> 1. **Reflection:** How thoughtfully did you analyze your own work and learning?

Reviewer: ```suggestion
1. **Reflection and growth:** Did you thoughtfully reflect on your learning and growth throughout the process?
```
